### DISCUSSION TITLE
How to access enableRowSelection property after intializing the table?

### DISCUSSION THREAD

**Discussion by c-semaan** [TanStack/table#4976] [General] [answered]
```
  const table = useReactTable({
    data,
    columns,
    enableRowSelection: canRowSelection, // how do I access this property from the table?
    state: {
      sorting,
      columnVisibility,
      rowSelection,
      columnFilters,
    },
  });
```

Ideally I want to access the row selection state from the table directly without having to pass down props to other components but I cannot find the accessor property for it. Is the accessory method missing?

**Comment by lucapollani** [comment, marked as answer]
I don't know whether is the right way but I think you can access it through the `table.options` property. 
https://tanstack.com/table/v8/docs/api/core/table#options-1
